vms/proposervm/block.go

```diff
@@ -254,6 +264,12 @@ func (p *postForkCommonComponents) buildChild(
 		)
 	}
 	if err != nil {
+		p.vm.ctx.Log.Error("unexpected build block failure",
+			zap.String("reason", "failed to generate proposervm block header"),
+			zap.Stringer("parentID", parentID),
+			zap.Stringer("blkID", innerBlock.ID()),
+			zap.Error(err),
+		)
```

Comment:
Erroring here technically won't cause an invariant break... However, it's unexpected so I felt like logging an error here still made sense.